Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -43,26 +43,32 @@ Wine.prototype.DXVK = function (dxvkVersion) {
         .to(this.prefixDirectory() + "/TMP/")
         .extract();
 
+    var forEach = Array.prototype.forEach;
+    var sys32dir = this.system32directory();
     var dxvkTmpDir = this.prefixDirectory() + "/TMP/dxvk-" + dxvkVersion;
+    var self = this;
 
-    cp(dxvkTmpDir + "/x32/d3d11.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/dxgi.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10core.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10.dll", this.system32directory());
-    cp(dxvkTmpDir + "/x32/d3d10_1.dll", this.system32directory());
+    //Copy 32 bits dll to system* and apply override
+    forEach.call(ls(dxvkTmpDir + "/x32"), function (file) {
+        if (file.endsWith(".dll")) {
+            cp(dxvkTmpDir + "/x32/" + file, sys32dir);
+            self.overrideDLL()
+                .set("native", [file])
```

Comment:
Should the `.dll` part of the filename be part of the override?

Reply:
Not a problem since overrideDLL check for that.